docs/editor/profiles.md

```diff
@@ -0,0 +1,169 @@
+
+You can also access the Profile command menu via the **Manage** gear button in the lower right of the Activity bar.
+
+![Create Profile command via the Manage button in the Activity bar](images/profiles/create-profile-via-manage.png)
+
+The **Profiles: Create Profile** command is also available in the Command Palette (`kb(workbench.action.showCommands)`).
+
+Once you choose whether to create a new profile based on the current profile or an empty profile, you are prompted to enter a name for the new profile.
+
+### Check the current profile
+
+The current profile name is displayed in several places in the VS Code UI:
+
+* Title bar
+* **File** > **Preferences** > **Profiles**
+* **Manage** gear button hover
+
+If you are still using the Default Profile, no profile name is displayed.
+
+The **Manage** gear button displays a badge with the first two letters of the active profile so you can quickly check which profile you are running.
+
+![Manage gear displaying "DE' to indicate that the user's 'Demo' profile is active](images/profiles/profile-gear-two-letters.png)
+
+### Edit a profile
+
+You can edit a profile just as you would normally change any VS Code configuration. You can install/uninstall/disable extensions, change settings, and adjust snippets like normal. These changes are stored in your currently active profile.
+
+### Workspace associations
+
+When you select create or select an existing profile, it is associated with the current workspace and whenever you open that folder, the workspace's profile is active. If you open another folder, the profile either goes back to the Default Profile if no profile is set, or switches to that folder's associated profile.
+
+Setting a workspace's profile back to the Default Profile clears the workspace's profile association.
+
+## Managing profiles
+
+### Switch profiles
+
+You can quickly switch between profiles with the **Profiles: Switch Profile** command in the Command Palette, which presents a dropdown listing your available profiles.
+
+### Rename a profile
+
+You can rename an existing profile via the **Rename** command in the Profiles menu.
+
+### Delete a profile
+
+You can delete a profile via the **Delete Profile** command. The **Delete Profile** drop down lets you select which profile(s) to delete.
+
+## Profile contents
+
+The **Profiles: Show Contents** command (available in the Command Palette or Profiles menus) brings up the Profiles view, where you can review the customizations for the profile.
+
+![Profile view displaying contents of a Doc Work profile](images/profiles/profile-show-contents.png)
+
+A profile can include:
+
+* Settings - In a profile-specific `settings.json` file.
+* Extensions - You can remove extensions from the profile by unchecking their entry in the Profiles view.
+* UI state - Currently open views and their positioning.
```

Comment:
UI state is a bit more than that. @sandy081 do you have good ideas how to generalise this sentence?
Maybe "Currently open views, their positioning and any other layout state."

Reply:
UI Layout includes - Views layout (positions), visibility of views and actions.

Reply:
Can users hide actions?

Reply:
Yes.